command/server.go

```diff
@@ -1732,6 +1759,36 @@ func (c *ServerCommand) removePidFile(pidPath string) error {
 	return os.Remove(pidPath)
 }
 
+func CheckMigration(b physical.Backend) (string, error) {
+	startTime := ""
+
+	entry, err := b.Get(context.Background(), migrationLock)
+
+	if err != nil {
+		return "", err
+	}
+
+	if entry != nil {
+		startTime = string(entry.Value)
+	}
+
+	return startTime, nil
+}
+
+func SetMigration(b physical.Backend, active bool) error {
+	if !active {
+		return b.Delete(context.Background(), migrationLock)
+	}
+
+	now := time.Now().Format(time.RFC3339)
```

Comment:
Could you make this a struct incase we want to add more data to it in the future?

Reply:
Done.